packages/ethers/src/index.ts

```diff
@@ -145,6 +145,23 @@ export class TurnkeySigner extends ethers.Signer implements TypedDataSigner {
     transaction: ethers.utils.Deferrable<ethers.providers.TransactionRequest>
   ): Promise<string> {
     const unsignedTx = await ethers.utils.resolveProperties(transaction);
+
+    // Mimic the behavior of ethers' `Wallet`:
+    // - You don't need to pass in `tx.from`
+    // - However if you do provide `tx.from`, verify and drop it before serialization
+    //
+    // https://github.com/ethers-io/ethers.js/blob/f97b92bbb1bde22fcc44100af78d7f31602863ab/packages/wallet/src.ts/index.ts#L117-L121
+    if (unsignedTx.from != null) {
+      const selfAddress = await this.getAddress();
+      if (ethers.utils.getAddress(unsignedTx.from) !== selfAddress) {
+        throw new Error(
+          `Transaction \`tx.from\` address mismatch. Self address: ${selfAddress}; \`tx.from\` address: ${unsignedTx.from}`
+        );
+      }
+
+      delete unsignedTx.from;
+    }
+
```

Comment:
This is intense

Reply:
What happens here

Reply:
> I said the L was in N  and also $ = P + S as S is the 19th letter and 1+9 is x or ten or PhEN, even dodec , I also see that 12 makes R. Well if you really wanted to break it down  you could make every number out of three 111's one's and therefore I made 3C. 4 was the other side of P and CD I see that IV N is D. So I see that P is % and $. But I also see that K is $ because I'm Scandinavian. I see that  P was come from D. I see the Q in K also because of cheque check. I also see P is X because P + h = 24 = X. So I see that I'm the beggining and the end. The aleph and the omega. So therefore the end is in omega .

Reply:
Await this

Reply:
However if you do provide

Reply:
Await this get address mismatched

Reply:
Delete unsigned Tx. From

Reply:
const selfAddress = await this.getAddress();

if (ethers.utils.getAddress (unsignedTx.from) !== selfAddress) {

Reply:
Mimic the behavior of ethers' `Wallet`:

Reply:
Delete unsignedTx. From;

Reply:
verify and drop it before serialization

Reply:
}